### ISSUE TITLE
Feature Suggestion : Light theme by default

### ISSUE THREAD

**Issue by julienreszka** [ProtonVPN/win-app#3]
```feature
In order to prevent my retina from adjusting when I switch windows
As a Proton VPN user that is using a light theme in every app because I live during the day and sleep at night (most people do this)
I want to be able to use a light theme

Scenario : Switching off dark theme
   Given I opened general settings
   When I switch "dark theme" off.
   Then I should see the theme change from dark to light.

Scenario : Switching on dark theme
   Given I opened general settings
   When I switch "dark theme" on.
   Then I should see the theme change from light to dark.
```

**Comment by sp1ritCS** [comment]
As mentioned on [Reddit](https://www.reddit.com/r/protonvpn/comments/etqr75) some users really like their dark theme and defend it.

Maybe a better Idea would be to detect if the user has Normal or Darkmtheme selected and setting the App Theme depending on this setting.

After a quick google search (not a windows user so idk) you can get the selected theme on the following path:
`HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Theme\Personalize/AppsUseLightTheme`

**Comment by julienreszka** [comment]
## Value proposition

-  In order to prevent my retina from adjusting when I switch windows
-  As a ProtonVPN user on windows 10
-  I want to be able to use a Proton VPN theme coherent with the rest of my system color settings

## Scenarios

### Scenario:  Auto Light theme

- Given my system settings are on Light Colors
- When I open the ProtonVpn client
- Then I should see the correct Light theme

### Scenario :  Auto Dark theme

  - Given my system settings are on Dark Colors
  - When I open the ProtonVpn client
  - Then I should see the Dark theme

**Comment by 0x4eadac43** [comment]
Just wandering if there is a light theme at all?

**Comment by Nihal247** [comment]
Hi @0x4eadac43, as of now, there is no light theme for ProtonVPN.

**Comment by cdfarrow** [comment]
Yes, to a light theme, please. It almost unusable in dark theme in my well-lit workspace. Apparently there's a world map there, but I can barely see it. 
A three-way choice is ideal: Light, Dark, Match system
